static/docs/api/frameworks/atomcore/annotated.html

```diff
@@ -20,7 +20,7 @@
   <td id="projectlogo"><img alt="Logo" src="/images/api-reference/api-ref-icon.png"/></td>
   <td id="projectalign" style="padding-left: 0.5em;">
    <div id="projectname">Open 3D Engine AtomCore API Reference
-   &#160;<span id="projectnumber">2111.2</span>
+   &#160;<span id="projectnumber">2210.0</span>
```

Comment:
Now that we've switched to a XX.XX scheme for the friendly name, this should probably be 22.10.0.